Alert/alert.css

```diff
@@ -0,0 +1,120 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body{
+    font-family: 'Poppins', sans-serif;
+    margin: 2rem;
+    color: #3e4152;
+}
+h1{
+    font-size:3rem;
+}
+
+h1, h2, p{
+    margin: 0.625rem 0;
+    line-height: 1.2;
+}
+
+h2{
+    font-size: 2rem;
+}
+p{
+    font-size: 1.2rem;
+    padding: 0.8rem;
+}
+.alert-box, .outlined-box, .filled-box{
+    border:1px solid #3e4152;
+    margin: auto;
+    width:80%;
+    padding: 0.3rem 1rem;
+    border-radius: 6px;
+    margin-bottom: 1.5rem;
+}
+.fa{
+    margin-left: 1rem;
+    font-size: 1.5rem;
+    padding: 0.4rem;
+}
+.alert{
+    display: flex;
+}
+span{
+    font-size: 1rem;
+    padding:0.4rem;
+}
+.success-alert,.warning-alert, .error-alert, .info-alert, .outline-success, .outline-warning, .outline-info, .outline-error, .filled-success,.filled-warning, .filled-error, .filled-info{
```

Comment:
rather than applying the same properties to different classes, create a single class and add this to HTML element. this CSS file looks better.